src/components/AppLayout/Sidebar/index.tsx

```diff
@@ -1,10 +1,14 @@
+import { lazy } from 'react'
 import styled from 'styled-components'
 import { Divider, IconText } from '@gnosis.pm/safe-react-components'
 
 import List, { ListItemType } from 'src/components/List'
 import SafeHeader from './SafeHeader'
 import DebugToggle from './DebugToggle'
 import { IS_PRODUCTION } from 'src/utils/constants'
+import { wrapInSuspense } from 'src/utils/wrapInSuspense'
+
+const DevTools = lazy(() => import('./DevTools'))
```

Comment:
I meant `if (IS_PRODUCTION) lazy(...)` so that it's not loaded on prod.

Reply:
How would you approach this? I'd assume we'd have to make a lazy-loader middleware component of sorts?

Reply:
I've tweaked it - it was very interesting to implement!